### ISSUE TITLE
dc-win2016 qwt stuck (because can't install .net framework without role)

### ISSUE THREAD

**Issue by 0rb677** [elliotkillick/qvm-create-windows-qube#13]
ping @elliotkillick Win7/Win10 works good. But there is a problems with Server versions. QWT stuck,  so no network on qube...any chance to fix?

![winserver](https://user-images.githubusercontent.com/60626673/82664908-65043700-9c3b-11ea-86c4-e37539b0544b.png)
![winserver2](https://user-images.githubusercontent.com/60626673/82664920-67669100-9c3b-11ea-94bc-6377dc5fab58.png)

**Comment by elliotkillick** [comment]
I have successfully reproduced this bug.

I don't think this issue has to do with that .NET error message though, that has been a known issue about QWT for a while and pops up on Windows 10 too. Just seems to be cosmetic. (It's already been fixed but QWT just needs to be rebuilt to include it.)

Instead, the issue seems to be related to `prepare-volume.exe` closing immediately after it's opened by the QWT installer causing the installer to (eventually, if you wait long enough) timeout waiting for the volume to be created and revert all the changes the it has made as a result of the failure.

At first, I thought this may have been due to how close after the boot of Windows `prepare-volume.exe` is executed (a race condition) so I added a big two minute `timeout` between these two lines:

https://github.com/elliotkillick/qvm-create-windows-qube/blob/81e47926dca0883cac53b3f0236e0b2362b5ed4d/tools-media/auto-qwt/run.bat#L14-L16

However, the issue still occurred.

The Qubes installer logs at `%TEMP%` didn't provide any hints and there were no logs at `C:\QubesLogs`.

Strangely enough, this bug does not always seem to occur. I've had a couple of times (with that command-line you provided) when the volume was made successfully and everything went on as normal.

Also, I found that if you just waited until after the installer reverted changes and then executed `E:\install-qwt.bat` or the installer directly then it would quite often succeed.

Note, this specific bug applies to just Windows Server 2016, not "all Server versions" (I also just tested Windows Server 2012R2 to make sure it was not occurring on there).

On Windows Server 2019 and Windows 10 there is a different issue where the volume just completely fails to be created. Already fixed in the pull request related to this issue: https://github.com/QubesOS/qubes-issues/issues/5090 (Also, noted on "QWT Known Issues" of README)

But perhaps that pull request will also fix this problem? I don't know because QWT needs to be rebuilt to include it.
